internal/controllers/replication/symphony.go

```diff
@@ -173,7 +177,9 @@ func (c *symphonyController) reconcileForward(ctx context.Context, symph *apiv1.
 		// Diff and update if needed when the composition for this synthesizer already exists
 		if existings, ok := existingBySynthName[variation.Synthesizer.Name]; ok {
 			existing := existings[0]
-			if equality.Semantic.DeepEqual(comp.Spec, existing.Spec) && equality.Semantic.DeepEqual(comp.Labels, existing.Labels) {
+			if equality.Semantic.DeepEqual(comp.Spec, existing.Spec) &&
+				equality.Semantic.DeepEqual(comp.Labels, existing.Labels) &&
+				equality.Semantic.DeepEqual(comp.Annotations, existing.Annotations) {
```

Comment:
Ahhh there was a reason why I didn't put this in the first time around, let me try to remember what it was

Reply:
We need to update that code path to set the annotation in the variation instead...